New package: Simplace v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: Simplace
- Repository: https://github.com/gk-crop/Simplace.jl
- Created by: @gk-crop
- Version: v0.0.1
- Commit: 42b0aee3da0b398fc23329a2c9368f8bd84e9b08
- Git reference: HEAD
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
Initial version
`````
<!-- END RELEASE NOTES -->

gk-crop commented:
Automerge complains about the package name, beeing similar to existing package names.
We want to keep the package name "Simplace" as it's a package that interacts with the simulation framework "Simplace" ([simplace.net](https://www.simplace.net/)) and there are [R](https://cran.r-project.org/package=simplace) and [Python](https://pypi.org/project/simplace/) package siblings named "simplace".

goerz commented:
I don't think the name is a problem, but this needs a bit more documentation before it can be registered. I don't think the existing ["Basic usage"](https://github.com/gk-crop/Simplace.jl?tab=readme-ov-file#basic-usage) is quite enough to actually get started. It would be good to extend this to a full example.

Or (in the longer term), a proper documentation with a tutorial and API docs. This isn't a requirement for the initial registration, but a README with enough information to allow someone to use the package is.

JuliaHubRegistrator committed:
New package: Simplace v0.0.1

UUID: b0b0a731-1897-4268-b4ff-e689c85594d9
Repo: https://github.com/gk-crop/Simplace.jl.git
Tree: 72c8ad9c4dfa21f04edf08e588a2f592222b7f68

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8

gk-crop commented:
Thanks for the suggestion. I added a full example to README.md.

gk-crop commented:
[noblock]

goerz commented:
> [noblock]

_All_ comments have to contain [noblock]
